mediagrains/gsf.py

```diff
@@ -704,16 +702,8 @@ def _write_rational(file, value):
     _write_uint(file, value.denominator, 4)
 
 
-def seekable(file):  # pragma: no cover
-    if PY3:
-        return file.seekable()
-    else:
-        try:
-            file.tell()
-        except IOError:
-            return False
-        else:
-            return True
+def seekable(file):
+    return file.seekable()
```

Comment:
Minor: maybe worth removing this function and replacing calls with `file.seekable`?